README.md

```diff
@@ -2,6 +2,9 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses/branch/main/graph/badge.svg?token=8mmcEy6jrV)](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses)
 
+Heroku QA Dashboard: https://dashboard.heroku.com/apps/s22-4pm-2-courses-qa
+Heroku QA Deployment: https://s22-4pm-2-courses-qa.herokuapp.com/
```

Comment:
I just tried accessing this and it looks like there's no frontend due to the missing production flag. You might want to add that in!